File: app/src/main/java/com/ebr163/inspector/sample/HomeActivity.java
package com.ebr163.inspector.sample;

import android.content.Intent;
import android.os.Bundle;

import com.ebr163.inspector.sample.pair.PairExampleActivity;
import com.ebr163.inspector.sample.value_example.ValueExampleActivity;
import com.ebr163.inspector.sample.view_example.ViewExampleActivity;

import androidx.appcompat.app.AppCompatActivity;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
        findViewById(R.id.viewExample).setOnClickListener(view -> startActivity(new Intent(HomeActivity.this, ViewExampleActivity.class)));
        findViewById(R.id.variableExample).setOnClickListener(view -> startActivity(new Intent(HomeActivity.this, ValueExampleActivity.class)));
        findViewById(R.id.pairExample).setOnClickListener(view -> startActivity(new Intent(HomeActivity.this, PairExampleActivity.class)));
    }
}

File: app/src/main/java/com/ebr163/inspector/sample/HomeActivity.kt
package com.ebr163.inspector.sample

import android.content.Intent
import android.os.Bundle

import com.ebr163.inspector.sample.pair_example.PairExampleActivity
import com.ebr163.inspector.sample.value_example.ValueExampleActivity
import com.ebr163.inspector.sample.view_example.ViewExampleActivity

import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_home.*

class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
        viewExample.setOnClickListener { startActivity(Intent(this@HomeActivity, ViewExampleActivity::class.java)) }
        variableExample.setOnClickListener { startActivity(Intent(this@HomeActivity, ValueExampleActivity::class.java)) }
        pairExample.setOnClickListener { startActivity(Intent(this@HomeActivity, PairExampleActivity::class.java)) }
    }
}